apps/www/_blog/2023-04-19-whats-new-in-pg-graphql-v1-2.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'What''s New in pg_graphql v1.2'
+description: 'New Features in the v1.2 release of pg_graphql'
+launchweek: 7
+tags:
+  - postgres
+  - graphql
+  - planetpg
+date: '2023-04-94'
+published_at: '2023-04-14T08:00:00.000-07:00'
+toc_depth: 3
+author: oli_rice
+image: launch-week-6/pggraphql/og-pg-graphql.png
+thumb: launch-week-6/pggraphql/og-pg-graphql.png
+---
+
+It's been 4 months since the [1.0.0 release of pg_graphql](https://supabase.com/blog/pg-graphql-v1).
+Since then, we’ve pushed several features to improve the APIs `pg_graphql` produces. 
+
+In this article, we’ll walk through those features and show examples of each.
+
+### View Support
+
+Prior to `v1.1` `pg_graphql` was restricted to reflecting standard tables. Following that release, views, materialized views, and foreign tables are similarly reflected in the GraphQL schema.
+
+```sql
+create view "ProjectOwner" as 
+  select
+    acc.id,
+    acc.name
+  from
+    account acc
+    join role r
+      on r.id = acc.role_id
+  where
+    acc.role = 'project_owner'
+```
+
+Since all entities exposed by pg_graphql require primary keys, we must define that constraint for the view using a comment directive
```

Comment:
```suggestion
Since all entities exposed by `pg_graphql` require primary keys, we must define that constraint for the view. We do that using a comment directive:
```